config/crd/patches/webhook_in_blockdeviceclaims.yaml

```diff
@@ -0,0 +1,14 @@
+# The following patch enables a conversion webhook for the CRD
```

Comment:
All the auto-generated configs are not needed here, we can remove things like webhooks